envoy-control-core/src/test/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/TestNodeFactory.kt

```diff
@@ -27,6 +28,10 @@ fun nodeV3(
         meta.putFields("service_name", string(serviceName))
     }
 
+    discoveryServiceName?.let {
+        meta.putFields("service_name", string(discoveryServiceName))
```

Comment:
Shouldn't it be `"discovery_service_name"`?

Reply:
done